R/R-nanoarrow/Portfile

```diff
@@ -3,18 +3,20 @@
 PortSystem          1.0
 PortGroup           R 1.0
 
-R.setup             cran apache nanoarrow 0.2.0.2
+R.setup             cran apache nanoarrow 0.3.0
 revision            0
 categories-append   devel
-maintainers         nomaintainer
-license             Apache-2
+maintainers         {@barracuda156 gmail.com:vital.had} openmaintainer
+license             Apache-2+
```

Comment:
this doesn't seem right - there isn't a higher version than 2 for the Apache license as far as I can tell

Reply:
It comes from here in upstream: https://github.com/apache/arrow-nanoarrow/blob/3952320d2d54d1af54db21ad688672984b45eb9c/r/DESCRIPTION#L19

CRAN just parses DESCRIPTION file, from what I understand, and I usually rely on CRAN page for license info, as long as the package is available on it.

Reply:
P. S. Technically, `2 >= 2` is a true statement anyway.

Reply:
thanks - we want to make sure we use licenses that MacPorts recognizes so that it can determine whether a port is distributable and/or licenses for dependents conflict.